src/Microsoft.Dnx.TestHost/Program.cs

```diff
@@ -32,13 +32,21 @@ public int Main(string[] args)
 
             var env = (IApplicationEnvironment)_services.GetService(typeof(IApplicationEnvironment));
 
-            var portOption = application.Option("--port", "Port number to listen for a connection.", CommandOptionType.SingleValue);
-            var projectOption = application.Option("--project", "Path to a project file.", CommandOptionType.SingleValue);
+            var portOption = application.Option(
+                "--port", "Port number to listen for a connection.", CommandOptionType.SingleValue);
```

Comment:
Thought this approach was something you mentioned a while back...like either have all params in one single line or if breaking one param, break other ones too?